src/Operators/finitedifference.jl

```diff
@@ -229,7 +229,7 @@ return_eltype(::FiniteDifferenceOperator, arg) = eltype(arg)
 @noinline invalid_boundary_condition_error(op_type::Type, bc_type::Type) =
     error("Boundary `$bc_type` is not supported for operator `$op_type`")
 
-boundary_width(op::FiniteDifferenceOperator, bc::BoundaryCondition) =
+boundary_width(op::FiniteDifferenceOperator, bc::BoundaryCondition, args...) =
```

Comment:
besides adding an argument to boundary width, most of the changes here seem unnecessary?  it makes it hard to review if we don't have minimal diffs.

Is there anything else materially that changed?

Reply:
Everything else was just formatting. Sorry, I found it a bit hard to read with the inconsistent formatting.

Reply:
Oh, I should note that stecil_interior_width was modified the same way as boundary_width, for the same reason.